### PR TITLE
Transfer Ownership

### PR DESCRIPTION
**Pull request by martinratinaud**

### CODE REVIEW

File: MIGRATING.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 First of all, thanks for taking the time to contribute! 🎉👍
 
+## 2022-05-04 Transfer to new organization
+
+Project has been moved from `ambanum/OpenTermsArchive-api` to `OpenTermsArchive/OpenTermsArchive-api`
```
Comment (MattiSG) [inline MIGRATING.md]:
```suggestion
Project has been moved from `ambanum/OpenTermsArchive-api` to `OpenTermsArchive/api`.
```

File: ops/roles/ota-api/github-clone/vars/main.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 app: "ota-api-{{ ENV }}"
-repository: "https://github.com/ambanum/OpenTermsArchive-api.git"
+repository: "https://github.com/OpenTermsArchive/OpenTermsArchive-api.git"
```
Comment (MattiSG) [inline ops/roles/ota-api/github-clone/vars/main.yml]:
```suggestion
repository: "https://github.com/OpenTermsArchive/api.git"
```

**Review by MattiSG — DISMISSED** [review]
Thanks! 👍

I'm in favour of not repeating OpenTermsArchive in both organisation and repository 🙂

**Comment by martinratinaud** [comment]
> Thanks! 👍
> 
> I'm in favour of not repeating OpenTermsArchive in both organisation and repository 🙂

I thought about it, but I think we should keep `api` for the real API we might build in the future.

What about `legacy-api` or `api-legacy` ? This way, we're not stuck with an old project having the url we want.

WDYT @MattiSG

**Comment by MattiSG** [comment]
Good point! Then I'm in favor of `api-legacy` 🙂

**Pull request merged**